micro-infra-hystrix/src/main/groovy/com/ofg/infrastructure/hystrix/IsHystrixServletEnabled.groovy

```diff
@@ -0,0 +1,20 @@
+package com.ofg.infrastructure.hystrix
+
+import com.ofg.infrastructure.config.PropertyAbsentOrEnabledCondition
+import groovy.transform.CompileStatic
+import groovy.transform.PackageScope
+
+/**
+ * Checks for the presence of the property to verify whether Hystrix is enabled or not
+ */
+@CompileStatic
+@PackageScope
+class IsHystrixServletEnabled extends PropertyAbsentOrEnabledCondition {
+
+    public static final String HYSTRIX_SERVLET_ENABLED = 'com.ofg.infra.microservice.hystrix.servlet'
```

Comment:
Damin, so it's already inconsistent all over the place: https://github.com/4finance/micro-infra-spring/blob/master/micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/metrics/config/IsGraphitePublishingEnabled.groovy#L12